### DISCUSSION TITLE
Solo V2 key id length (Keeper Password Manager integration)

### DISCUSSION THREAD

**Discussion by imilchev** [solokeys/solo2#109] [Q&A]
I have a SoloV2 key and I managed to register it for all services that I use BUT I also use Keeper as my password manager and there it fails. I had a call with their support and they saw what I was dealing with. What they did is order a SoloV1 key and with it the entire flow works.

The details I got back from their developers are:
```
I have received a SOLO key (https://solokeys.com/collections/all/products/solo-tap-usb-a-preorder) but it works just fine for me. The problem with the customer’s key is that it produces an extremely long key id 376 characters while my key does only 70. So this problem is with the specific key this customer has. You might ask him if he did some customizations over the key, and also the key exact model. Another thing to try is to use another solo key if he has one or reset his key to defaults to see if that works (he will need to re-register this key with any service used previously).
```

Can someone shed a little more light on this issue and why the V2 is generating a longer key id?

**Comment by imilchev** [comment]
Can someone from the maintainers answer this?

**Reply by nickray** [reply]
Yeah the key handles are long. If you don't mind losing NFC functionality, https://gist.github.com/nickray/7de5d40d5d93b7a1b6d0983062c9339e has a prerelease of a firmware with shorter key handles.

**Reply by mathiasrabe** [reply]
Is there a roadmap or something when these changes will be merged into main branch?
